### ISSUE TITLE
We lost speech kitchen.org

### ISSUE THREAD

**Issue by fmetze** [srvk/srvk.github.io#1]
We need to clean up all references to speechkitchen.org in the Github repository, the srvk.github.io web page, and the speech-kitchen.org web site.

**Comment by riebling** [comment]
The speech-kitchen.org web site changes prove to be 'interesting' due to what I'm assuming are WordPress quirks. Might require digging into the WordPress database to fully achieve the change. (I spent some time trying to fix through the WordPress admin page [ http://speech-kitchen.org/wp-admin ]  at the time we lost the domain name, and it's true things are not yet fully ported to the new URL. This is because much of the site is stored in a MySQL database, and requires supervised global search/replace.

Symptoms:
1. when visiting speech-kitchen.org, the page that gets rendered seems to be missing the graphic logo at the top.

So there's more to be done making speech-kitchen.org site fully reachable and functional again. I have installed a WordPress plugin that assists in the search/replace part of this, will update this ticket with progress.
